### ISSUE TITLE
Login not Working 

### ISSUE THREAD

**Issue by younasm** [cgjacklin/Neu-Spitfire-e-commerce-platform#8]
I downloaded all the dependencies on the front-end and back-end and now there is no issue both servers are running but still, I am not able to login I tried several times to login neither it's login nor shows the error.

**Comment by cgjacklin** [comment]
The database is already removed. You may create one or it won't work.

**Comment by younasm** [comment]
Yes, I have created one from your given SQL but when I click on the login button it neither shows any error nor its login. Can you give me the solution?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
